notifier/management/commands/forums_digest_flagged.py

```diff
@@ -0,0 +1,75 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from optparse import make_option
+
+from notifier.digest import render_digest_flagged
+from notifier.pull import get_flagged_threads
+from notifier.tasks import do_forums_digests_flagged
+from notifier.user import get_moderators
+
+
+class Command(BaseCommand):
+    """
+    This Command is used to send a digest of threads with flagged posts to
+    forum moderators.
+    """
+
+    help = "Send a digest list of threads with flagged posts to each moderator"
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--is-dry-run',
+            action='store_true',
+            dest='is_dry_run',
+            default=False,
+            help='for each course with flagged posts, display the email address' +
+                 ' for each recipient, and the rendered email text WITHOUT' +
+                 ' sending a message',
+        ),
+    )
+
+    def handle(self, *args, **options):
+        """
+        Handle a request to send a digest of threads with flagged posts to forum
+        moderators.
+        """
+        is_dry_run = options.get('is_dry_run')
+
+        if is_dry_run:
+            course_threads = {}
+            for thread in get_flagged_threads():
+                course_id = thread.get('course_id')
+                commentable_id = thread.get('commentable_id')
+                comment_thread_id = thread.get('comment_thread_id')
+                if course_id and commentable_id and comment_thread_id:
+                    if not course_threads.has_key(course_id):
+                        course_threads[course_id] = []
+                    thread_url = '{}/courses/{}/discussion/forum/{}/threads/{}'.format(
+                        settings.LMS_URL_BASE, course_id, commentable_id, comment_thread_id
+                    )
+                    course_threads[course_id].append(thread_url)
+
+            for course_id in course_threads:
+                moderators = get_moderators(course_id)
+                try:
+                    user = moderators.next()
```

Comment:
Oh I only want to print the first rendered body since it's the same for all users. I put all the API calls before the prints because I wanted to avoid having log statements interspersed with the content you care about